FrontEnd/src/components/ProfileView/ProfileNavBar/ProfileNavBar.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import classes from "./ProfileNavBar.module.css";
+
+class ProfileNavBar extends Component {
```

Comment:
Is there any need to use class components? It's better to use same components type.